src/magnify/registry.py

```diff
@@ -29,11 +32,254 @@ def component_factory(*args, **kwargs):
     return component_decorator
 
 
-def mini_chip(**kwargs):
+def mini_chip(
+    data: ArrayLike | str,
+    times: Sequence[int] | None = None,
+    channels: Sequence[str] | None = None,
+    debug: bool = False,
+    shape: tuple[int, int] = (8, 8),
+    pinlist: str | None = None,
+    blank: str | list[str] | None = None,
+    overlap: int = 102,
+    rotation: int = 0,
+    row_dist: float = 375 / 1.61,
+    col_dist: float = 400 / 1.61,
+    min_button_radius: int = 4,
+    max_button_radius: int = 15,
+    low_edge_quantile: float = 0.1,
+    high_edge_quantile: float = 0.9,
+    num_iter: int = 5000000,
+    min_roundness: float = 0.2,
+    cluster_penalty: float = 50,
+    roi_length: int = 61,
+    progress_bar: bool = False,
+    search_timestep: list[int] | None = None,
```

Comment:
This can be an individual int as well.

Reply:
Just to confirm, are we referring to "search_timestep"? :)

Reply:
search_timestep

Reply:
Resolved